modules/nodes-descheduler-strategies.adoc

```diff
@@ -42,3 +44,9 @@ The `RemovePodsHavingTooManyRestarts` strategy ensures that Pods that have been
 This situation could occur if a Pod is scheduled on a node that is unable to start it. For example, if the node is having network issues and is unable to mount a networked persistent volume, then the Pod should be evicted so that it can be scheduled on another node. Another example is if the Pod is crashlooping.
 +
 This strategy has two configurable parameters: `PodRestartThreshold` and `IncludingInitContainers`. If a Pod is restarted more than the configured `PodRestartThreshold` value, then the Pod is evicted. You can use the `IncludingInitContainers` parameter to specify whether restarts for Init Containers should be calculated into the `PodRestartThreshold` value.
+
+Pod life time::
+
+The `PodLifeTime` strategy evicts Pods that are too old.
++
+After a Pod reaches the age, in seconds, set by the `MaxPodLifeTimeSeconds` parameter, it should be evicted.
```

Comment:
Can something stronger be used here other than `it should be evicted`? Maybe `it is evicted.`?